common/lib/capa/capa/responsetypes.py

```diff
@@ -1597,9 +1597,8 @@ def get_score(self, student_answers):
             student_float = evaluator({}, {}, student_answer)
         except UndefinedVariable as undef_var:
             raise StudentInputError(
-                _(u"You may not use variables ({bad_variables}) in numerical problems.").format(
-                    bad_variables=undef_var.message,
-                )
+                _(u"Answers can include numerals, operation signs, and a few specific characters, "
```

Comment:
I don't see why it would not get scraped.  

In this particular case because both usages are in separate Classes with a thousand or more lines between them I am not sure there is value in moving them into a variable.